src/pg/test/db/dbWrite.test.js

```diff
@@ -191,4 +208,22 @@ describe('postgres', () => {
       RETURNING *, "id" AS "$key", current_timestamp AS "$ver"`;
     expectSql(mockQuery.mock.calls[0][0], sqlQuery);
   });
+
+  test('jsonb_null_patch_uses_plain_null', async () => {
+    await store.write('sfWrite.foo', {
+      write: { WhatId: { $val: null } },
+    });
+
+    const sqlQuery = sql`
+      UPDATE "sfWrite" SET
+        "write" = case jsonb_typeof("write")
+          when 'object' then "write"
+          else '{}'::jsonb
+        end || jsonb_build_object(${'WhatId'}::text, ${'{"WhatId":null}'}::jsonb),
```

Comment:
This is not correct, the expectation is:


```suggestion
        end || jsonb_build_object(${'WhatId'}::text, ${'{"$val":null}'}::jsonb),
```